### ISSUE TITLE
BUG:  Fov increase problem

### ISSUE THREAD

**Issue by cengizalcan** [3a1/Zodiak#7]
This line count change to 1, fix this problem.

https://github.com/3a1/Zodiak/blob/9b7e2160c3df286ed05be5c722a55f2ee920bd34/src/Driver/game.c#L313

**Comment by 3a1** [comment]
yeah, my typo, thx for your attention to it. problem exactly in this like:
`if (count == 0) `
changed to 
`if (count == 1) `

fixed :)

**Comment by cengizalcan** [comment]
Welcome brother. I adding some more features too. Thank you soo much for this cheat. :)))
